content/ember/angle-brackets.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Use angle bracket component syntax for all new components
+language: ember
+---
+
+But don't use the new syntax for components that already exists with the old syntax, i.e one syntax per component
+
+```handlebars
+  {{! Good }}
+  <EditPage @page={{page}} class="u-display--flex" />
```

Comment:
should it be `class` or `@class`?

Reply:
regular html attributes is defined without `@`

Reply:
`class` since it's an HTML attribute and not a property passed to the component

Reply:
That is one of the benefits with angle brackets

Reply:
Nice!